Neos.Neos/Tests/Behavior/Features/NeosHistory/Entities/AccountsUsers.feature

```diff
@@ -5,7 +5,7 @@ Feature: Accounts / User Entity Monitoring
   Background:
     Given I have an empty history
     Given I have the following "monitorEntities" configuration:
-    """
+    """yaml
```

Comment:
What's the purpose here?

Reply:
provide ide highlighting for the code fence ;)

Reply:
Ok… and Behat doesn't care? Interesting… TIL.

Reply:
yes this is so cool ^^ and we use it also for the new escr tests :D